.github/workflows/fly.yml

```diff
@@ -0,0 +1,14 @@
+name: Fly Deploy
+on: [push]
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+jobs:
+  deploy:
+    name: Deploy app
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: npm ci
```

Comment:
caching?

Reply:
Fixed in [dc82911](https://github.com/Secret333Boy/node.js.labs/pull/3/commits/dc829115da8c3c0099c3edb8c84c6befa520363b)